src/puppetlabs/puppetdb/admin.clj

```diff
@@ -1,6 +1,8 @@
 (ns puppetlabs.puppetdb.admin
   (:require [compojure.core :as compojure]
-            [puppetlabs.i18n.core :as i18n]
+            [puppetlabs.i18n.core :refer [trs tru]]
```

Comment:
I've been trying to stick to `i18n/trs` and `i18n/tru` to make the grep a little easier fwiw